E2E/specs/example.spec.js

```diff
@@ -0,0 +1,16 @@
+describe("test set", () => {
+  it("should open website", () => {
+    browser.get("http://10.0.75.1:9000/#/");
+    const el1 = element(by.css(".new-todo"));
+    const header = element(by.css("h1"));
+    const headerText = header.getText();
+    expect("Todos").toEqual(headerText);
+    el1.sendKeys("Test", protractor.Key.ENTER);
+    browser.sleep(1000);
```

Comment:
This is an Angular application - if possible try to user browser.waitForAngular() instead of browser.sleep()

Reply:
You are god damn right.